entregas/vaqueroSara/PiedraPapelTijera.java

```diff
@@ -0,0 +1,48 @@
+import java.util.Scanner;
+
+public class PiedraPapelTijera.java {
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+
+        String piedra = "piedra";
+        String tijera = "tijera";
+        String papel = "papel";
+
+        System.out.println("Bienvenido a piedra, papel o tijera.");
+        System.out.println(
+                "Escoge piedra, papel o tijera para jugar contra la computadora. Por favor, introduce la palabra exacta");
+        String userValue = scanner.nextLine().toLowerCase();
+
+        String validOption = (userValue.equals(piedra) || userValue.equals(tijera) || userValue.equals(papel))
+                ? userValue
+                : "inválido";
+
+        System.out.println(validOption.equals("inválido") ? "Por favor, reinicia e introduce un valor válido" : "");
+
+        int maximo = 3;
+        int minimo = 1;
+
+        int compValue = (int) (Math.random() * (maximo - minimo) + minimo);
```

Comment:
```suggestion
        int computerValue = (int) (Math.random() * (maximo - minimo) + minimo);
```

He tenido que leer dos veces para enterarme que era computer y no comparation.